src/renderer/reducers/settings.js

```diff
@@ -68,9 +75,12 @@ export type SettingsState = {
   loaded: boolean, // is the settings loaded from db (it not we don't save them)
   hasCompletedOnboarding: boolean,
   counterValue: string,
+  // TODO: DEPRECATE THESE
   preferredDeviceModel: DeviceModelId,
```

Comment:
to me this one is the only one that is not deprecated. I would keep this one and only this one as discussed yesterday.
preferredDeviceModel would "move" independently of the other DeviceModelInfo.

preferredDeviceModel can be refreshed very often, because we quickly can know what is the model plugged. on the other end, the apps + deviceInfo is a costy info to get so we only refresh it in a few cases (entering manager essentially)